docker-compose.yml

```diff
@@ -32,7 +32,7 @@ services:
       - discovery.type=single-node
       - network.host=0.0.0.0 # is this still needed? 
     volumes:
-      - /Users/gh/zeeguu/endpoints/dev_data_folder/elasticsearch_db/data:/usr/share/elasticsearch/data
+      - ./Users/gh/zeeguu/endpoints/dev_data_folder/elasticsearch_db/data:/usr/share/elasticsearch/data
```

Comment:
I added the ./ to avoid running into an error when launching the elasticsearch service (otherwise for me it just breaks by attempting to create a directory with mkdir. Using this ./ seems to solve that issue. I can remove this and just try to keep this locally or maybe a better solution would to make it a relative path?

Reply:
added issue for this for now: https://github.com/zeeguu/api/issues/106